addons/rose/addon/styles/rose/components/_icon.scss

```diff
@@ -17,3 +17,19 @@
     vertical-align: text-top;
   }
 }
+
+.is-loading .rose-icon {
```

Comment:
I'm thinking this is not well-scoped enough to live here.  Can we move this into the applications for now?  It's sort of temporary until we fully adopt HDS.

Reply:
Absolutely. I will move it to applications.

Reply:
Fixed.